packages/manager/modules/vps/src/routing.js

```diff
@@ -56,6 +65,8 @@ export default /* @ngInject */ ($stateProvider) => {
           }
           return promise;
         },
+        isVpsNewRange: /* @ngInject */ (stateVps) =>
+          stateVps.model.version === NEW_RANGE_VERSION,
```

Comment:
Another way could have been to have a `Vps` class and manipulate instances with a `isNewRange()` method

Reply:
I'll keep this in mind for a later refactor or changes on vps section